determined-ai/determined
# feat: Flat Run Actions

## Ticket

[ET-110]


## Description

Flat Run Actions
`Tensorboard` and `Pause/Unpause` actions are out of this PR, so not included
`Move` action will be implemented after #9390 is merged into this

## Test Plan

- Enable Run table
- Check if action button is visible when one or more flat runs are selected
- Check if `archive`, `unarchive`, `delete`, `kill` actions are working with/without RBAC
- If runs belong to searches, the warning modal should show up. if user cancel it, move action won't happen. 


## Checklist

  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.




[ET-110]: https://hpe-aiatscale.atlassian.net/browse/ET-110?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DET-123]: https://hpe-aiatscale.atlassian.net/browse/DET-123?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
